tts-deploy-on-embedded.ipynb

```diff
@@ -0,0 +1,280 @@
+    "\n",
+    "The RMIR generated in this tutorial can be deployed using [riva_quickstart](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/quick-start-guide.html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14e5a5e4",
+   "metadata": {},
+   "source": [
+    "import pathlib to get the paths"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "34284f33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pathlib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "670aea87",
+   "metadata": {},
+   "source": [
+    "### Download models\n",
+    "We will download pretrained [Fastpitch](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa) and [HifiGan](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa) models from ngc. You can replace these models with the paths of your custom model, incase of custom models."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eca2a574",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_fastpitch_ipa:deployable_v1.0\"\n",
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_hifigan_ipa:deployable_v1.0\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d93f949e",
+   "metadata": {},
+   "source": [
+    "Download the riva model repository files from ngc."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "51b93e9f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!ngc registry model download-version \"nvidia/tao/speechsynthesis_en_us_auxiliary_files:deployable_v1.3\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "19d3303d",
+   "metadata": {},
+   "source": [
+    "### Set configs and params.\n",
+    "Set following config parameters:  \n",
+    "`acoustic_model`: Full path for acoustic_model.riva file from [ngc](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_fastpitch_ipa)  \n",
+    "`vocoder`: Full path for vocoder.riva file file from [ngc](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechsynthesis_en_us_hifigan_ipa)  \n",
+    "`out_dir`: Directory to put TTS.rmir file. The RMIR will be placed in ${out_dir}/RMIR/RMIR_NAME.rmir  \n",
+    "`voice`: Set the voice name of the model.  \n",
+    "`key`: This is the encryption key used in nemo2riva. Same key will be used to deploy the RMIR generated in this tutorial.  \n",
+    "`use_ipa`: Set to \"y\" or \"Y\" if the model uses IPA phones, \"no\" if the model uses arpabet.  \n",
+    "`lang`: Model language.  \n",
+    "`sample_rate`: Sample rate of generated audios.  \n",
+    "`machine_type`: type of machine the tutorial is being run on. Acceptable values are `arm` and `amd`.  "
```

Comment:
Move this to the beginning of the document. This should be right before you download the .riva files